pybikes/data/cocacola.json

```diff
@@ -0,0 +1,33 @@
+{
+    "instances": [
+        {
+            "tag": "cork",
+            "meta": {
+                "latitude": 51.893604,
+                "longitude": -8.494174,
+                "city": "Cork",
+                "country": "IE"
+            }
+        },
+        {
+            "tag": "limerick",
+            "meta": {
+                "latitude": 52.668746,
+                "longitude": -8.627809,
+                "city": "Limerick",
+                "country": "IE"
+            }
+        },
+        {
+            "tag": "galway",
```

Comment:
You can either set it on the meta as `system` in `bikeshare_ie.py` or as name in `data/bikeshare_ie.json`.